lib/linalg.dx

```diff
@@ -16,6 +15,8 @@ def refl_more {n} [Ix n] (i:n) : (i..) = unsafe_project _ i
 def upper_tri_diag {n v} (u:UpperTriMat n v) : n=>v = view i. u.i.(refl_more i)
 def lower_tri_diag {n v} (l:LowerTriMat n v) : n=>v = view i. l.i.(refl_less i)
 
+def is_last {n} [Ix n] (x:n) : Bool = ((ordinal x) + 1) == size n
```

Comment:
Nit: This should work without parens at all: `ordinal x + 1 == size n`, though it's up to you which form you consider the most readable.

Reply:
Actually it does require the outer parens, but I got rid of the inner ones, thanks.